include/aie/Dialect/AIEVec/IR/AIEVecLLVMIntrOp.td

```diff
@@ -46,6 +50,14 @@ def MacConfAcc32IntrOp :
                    VectorOfLengthAndType<[16], [I64]>:$acc,
                    I32:$conf)>;
 
+def MacConfAcc64IntrOp :
+    AIEVec2_IntrOp<"I512.I512.ACC1024.acc64.mac.conf",
+        [TypeIs<"res", VectorOfLengthAndType<[16], [I64]>>]>,
+    Arguments<(ins VectorOfLengthAndType<[64], [I8]>:$lhs,
+                   VectorOfLengthAndType<[16], [I32]>:$rhs,
+                   VectorOfLengthAndType<[16], [I64]>:$acc,
```

Comment:
Maybe a comment about why accumulators are represented as I64 would be in place?

Reply:
The same would apply to all the types. The explanation would make references to an internal project, so I'm being cautious here.